content/docs/cookbooks/tls-auth.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Securing Prometheus endpoints using TLS authentication
+sort_rank: 1
+---
+
+# Securing Prometheus endpoints using TLS auth
+
+Prometheus does *not* directly support authentication via [Transport Layer Security](https://en.wikipedia.org/wiki/Transport_Layer_Security), and this is so [by design](/docs/introduction/faq/#tls). In fact, Prometheus provides no server-side authentication, authorization, or encryption.
+
+Nonetheless, we do strongly recommended that you use TLS or some other form of authentication when using Prometheus by applying it via reverse proxy. You can use any proxy you like with Prometheus. In this cookbook, however, we'll cover only [nginx](#nginx).
```

Comment:
I don't believe we make any recommendations in this area. TLS is also not authentication.